doc/api/fs.md

```diff
@@ -446,14 +446,22 @@ Reads data from the file and stores that in the given buffer.
 If the file is not modified concurrently, the end-of-file is reached when the
 number of bytes read is zero.
 
-#### `filehandle.readableWebStream()`
+#### `filehandle.readableWebStream(options)`
 
 <!-- YAML
 added: v17.0.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/46933
+    description: Added option to create a 'bytes' stream.
 -->
 
 > Stability: 1 - Experimental
 
+* `options` {Object}
+  * `type` {string|undefined} Whether to open a normal or a `'bytes'` stream.
+    **Default:** `undefined`
```

Comment:
Why do this need to be optional?

Reply:
Normal ReadableStream cannot be made into 'byob' mode the ReadableStream constructor needs the `type: 'bytes'` to open a bytes stream, also the logic with which we pull content changes hence added it as a option

Reply:
>Normal ReadableStream cannot be made into 'byob' mode the ReadableStream constructor needs the type: 'bytes' to open a bytes stream

Sure, I mean why pass `type: bytes` only optionally?

> also the logic with which we pull content changes hence added it as a option

But will it be visible to users?

Reply:
> Sure, I mean why pass type: bytes only optionally?

As in you want bytes to be the default way in which streams would open? Didn't do that lest it break something wanted to allow both the logic to stay on for now, maybe we can update that in the future!

> But will it be visible to users?

I sure hope so, atleast for most users using IDEs etc they should be able to know about the option I hope

Reply:
>Didn't do that lest it break something wanted to allow both the logic to stay on for now, maybe we can update that in the future!

This theoretically should not affect existing uses for non-byob mode, but okay.